lib/FallbackComponent.astro

```diff
@@ -1,5 +1,5 @@
 ---
-import type { SbBlokData } from "./types";
+import type { SbBlokData } from "./dist/types";
```

Comment:
I have changed types import to be `./dist/types` as per following how Astro team does that. See https://github.com/withastro/astro/blob/main/packages/astro-prism/Prism.astro#L2 as an example.

Basically `.astro` is meant to target production, as those files are basically copied-over to package.